react/week3/my-app/src/ApiFetch.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from "react";
+import ListOfResult from "./ListOfResult";
+
+const ApiFetch = () => {
+  const [subscribedState, setSubscribedState] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      const result = await fetch(
+        "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw"
+      ).then((res) => res.json());
+
+      console.log(result);
```

Comment:
Always remember to remove console log when you do final push of code, its not good practice to push console.log in Production.